### PR TITLE
Fix: Copy'n'Paste

### LINKED ISSUE #1571: Copy from Copy & Paste is broken.
Because selection items & order are unnecessary and confusing.

### PR DESCRIPTION
**Pull request by neotyk**
Closes #1571

### CODE REVIEW

File: src/cljs/athens/events.cljs [hunk 1]
```diff
@@ -383,55 +345,38 @@
...
 ;; using a set or a hash map, we would need a secondary editing/uid to maintain the head/tail position
 ;; this would let us know if the operation is additive or subtractive
 (reg-event-db
   :selected/down
   (fn [db [_ selected-items]]
-    (assoc-in db [:selection :items] (select-down selected-items))))
+    (let [last-item         (last selected-items)
+          next-block-uid    (db/next-block-uid last-item true)
+          ordered-selection (-> (into [] selected-items)
+                                (into [next-block-uid]))]
```
Comment (filipesilva) [inline src/cljs/athens/events.cljs:375]: Isn't this the same as `(into [next-block-uid] selected-items)`?

**Review by filipesilva — APPROVED** [review]
lgtm, 'cept all the failing style stuff

File: src/cljs/athens/events.cljs (hunk 1 above)
Comment (neotyk) [reply src/cljs/athens/events.cljs:375]: It's more like `(into (into [] selected-items) [next-block-uid])`

Comment (filipesilva) [reply src/cljs/athens/events.cljs:375]: Yes I misread the threading operator, thought it was a thread last.

**Pull request merged**